moderngl_window/capture/base.py

```diff
@@ -21,28 +22,28 @@ class BaseVideoCapture:
 
     def __init__(
         self,
-        source: Union[moderngl.Texture, moderngl.Framebuffer] = None,
```

Comment:
A source has to be defined, so treat it as such

Reply:
Definitely a good idea